src/xlsx/conversion/figure_conversion.ts

```diff
@@ -23,27 +24,30 @@ function convertFigure(
   id: string,
   sheetData: XLSXWorksheet
 ): FigureData<any> | undefined {
-  const x1 =
-    getColPosition(figure.anchors[0].col, sheetData) +
-    convertEMUToDotValue(figure.anchors[0].colOffset);
-  const x2 =
-    getColPosition(figure.anchors[1].col, sheetData) +
-    convertEMUToDotValue(figure.anchors[1].colOffset);
-
-  const y1 =
-    getRowPosition(figure.anchors[0].row, sheetData) +
-    convertEMUToDotValue(figure.anchors[0].rowOffset);
-  const y2 =
-    getRowPosition(figure.anchors[1].row, sheetData) +
-    convertEMUToDotValue(figure.anchors[1].rowOffset);
-
+  let x1: number, y1: number;
+  let height: number, width: number;
+  if (figure.figureSize) {
+    // one cell anchor
```

Comment:
`extractImage` can return a size and 2 anchors. Test the length of `figure.anchors` instead